src/lib/Scenes/Collection/Screens/__tests__/__snapshots__/CollectionArtworks-tests.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Collection Artworks renders properly 1`] = `"<rctscrollview scrolleventthrottle=\\"50\\" accessibilitylabel=\\"Artworks ScrollView\\"><view><view px=\\"0\\"><view style=\\"flex-direction: row;\\" accessibilitylabel=\\"Artworks Content View\\"></view></view></view></rctscrollview>"`;
```

Comment:
What Steve and I were aiming for in the Featured Artists component might be a good pattern https://github.com/artsy/emission/blob/849ab02f636ba54f51f32f7f00bd9e21899a62e1/src/lib/Scenes/Collection/Components/__tests__/FeaturedArtists-tests.tsx

Reply:
I updated the Collections (parent) container to use a snapshot of the entire view so I'm disabling the Collections Artworks snapshots now just to unblock this PR and get it merged. Most likely we don't need a snapshot test here because we're handling that in the Collection container.